src/atoms/affine/stack.jl

```diff
@@ -112,21 +112,25 @@ end
 
 # * `Value` is not owned by Convex.jl
 # * splatting creates zero-argument functions, which again are not owned by Convex.jl
+Base.hcat(args::AbstractExpr...) = HcatAtom(args...)
 
-hcat(args::AbstractExpr...) = HcatAtom(args...)
-function hcat(args::AbstractExprOrValue...)
+function Base.hcat(args::AbstractExprOrValue...)
+    if all(Base.Fix2(isa, Value), args)
```

Comment:
Should we try removing these `if` and see if the tests break?

Reply:
To me it feels safer to have it, re- your comment about something changing in Base. But I guess having it could mask issues if we’d rather it start erroring should something change 🤔

Reply:
I guess either approach is okay.